code/modules/paperwork/paper.dm

```diff
@@ -102,8 +102,7 @@
 		return ..()
 
 /obj/item/paper/proc/rename(mob/user)
-
-	if(HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
+	if(user && HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
```

Comment:
Actually, my bad, this user check should be an early return, since the rest of the proc uses user too

Reply:
Removed the check, it was broken even more deeply and GDN would fix it